README.rst

```diff
@@ -240,6 +240,59 @@ inside the current folder. If you omit the final ``tmp/`` you'll leave the
 empty ``tmp`` directory in place. (**WARNING!** Don't put the ``.`` *after*
 the ``--erase`` option! Obviously, your project files will all be deleted.)
 
+🏷️ Git-clean integration
+--------------------------
+
+PyClean integrates with Git's built-in ``git clean`` command to remove
+untracked files from your working directory. This provides a user-friendly
+way to automatically clean up files that are not under version control.
+
+**Requirements:** Git must be installed on your system for this feature to
+be available.
+
+The ``--git-clean`` (or ``-g``) flag runs ``git clean`` after all other
+cleanup operations (bytecode, debris, and free-form targets). The flag
+honors PyClean's ``--ignore``, ``--dry-run``, and ``--yes`` options:
+
+- **Interactive mode (default):** Git prompts you to confirm which files
+  to delete.
+- **Dry-run mode (--dry-run):** Shows which files would be deleted without
+  actually removing them.
+- **Force mode (--yes):** Deletes all untracked files without prompting.
+
+*Example:* Preview untracked files that would be removed:
+
+.. code:: shell
+
+    pyclean . --git-clean --dry-run
+
+*Example:* Interactively clean untracked files (Git will ask for confirmation):
+
+.. code:: shell
+
+    pyclean . --git-clean
+
+*Example:* Force deletion of all untracked files (use with caution!):
+
+.. code:: shell
+
+    pyclean . --git-clean --yes
+
+*Example:* Clean up everything (bytecode, debris, and untracked files):
+
+.. code:: shell
+
+    pyclean . --debris --git-clean --yes
```

Comment:
This doesn't match the style of the other sections. "Example:" is not used anywhere else. Make it match the style of the existing content.